projects/packages/jetpack-mu-wpcom/src/features/domain-email-nag/domain-email-nag.php

```diff
@@ -32,47 +32,69 @@ function get_account_url() {
 }
 
 /**
- * Find the domain, if any, that has an unverified email address.
+ * Get the current domain for the site.
  */
-function get_domain_with_unverified_email() {
-	if ( ! class_exists( 'Domain_Management' ) ) {
-		return false;
-	}
-
-	$domains = \Domain_Management::get_paid_domains_with_icann_verification_status();
+function get_domain() {
+	$urlparts = wp_parse_url( home_url() );
+	return $urlparts['host'];
+}
 
-	foreach ( $domains as $domain ) {
-		if ( $domain['is_pending_icann_verification'] === true ) {
-			return $domain['domain'];
-		}
-	}
-	return false;
+/**
+ * Checks if the site is using a custom domain.
+ */
+function is_custom_domain() {
+	$domain = get_domain();
+	return ! str_ends_with( $domain, 'wpcomstaging.com' ) && ! str_ends_with( $domain, 'wordpress.com' );
 }
 
 /**
  * Decides whether to render to the email nag.
  */
 function domain_email_nag() {
-	if ( ! should_show_domain_frontend_email_nag() ) {
+	if ( ! is_on_frontend_and_logged_in() && ! is_custom_domain() ) {
 		return;
 	}
 
-	$domain = get_domain_with_unverified_email();
+	wp_enqueue_style( 'wpcom-domain-email-nag-style', plugins_url( 'domain-nag.style.css', __FILE__ ), array(), Jetpack_Mu_Wpcom::PACKAGE_VERSION );
 
-	if ( ! $domain ) {
-		return;
-	}
+	$blog_id = Jetpack_Options::get_option( 'id' );
```

Comment:
This one's causing an error on the front end
![CleanShot 2023-11-09 at 20 10 55@2x](https://github.com/Automattic/jetpack/assets/1500769/b6c7cd7b-bfe1-4538-9b55-8432fbff3259)

I bet it's a missing namespace.